public/js/src/game/ui/sideEffects.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
this shouldn't be needed bcuz you have 2 exports, right?

Reply:
Nice catch - leftover from development. Will remove